run_brer/run_config.py

```diff
@@ -369,12 +369,15 @@ def __train(self, tpr_file=None, **kwargs):
             current_alpha = context.potentials[i].alpha
             # noinspection PyUnresolvedReferences
             current_target = context.potentials[i].target
+            current_stop_called = context.potentials[i].stop_called
 
             self.run_data.set(name=current_name, alpha=current_alpha)
             self.run_data.set(name=current_name, target=current_target)
-            self._logger.info("Plugin {}: alpha = {}, target = {}".format(current_name,
-                                                                          current_alpha,
-                                                                          current_target))
+            self.run_data.set(name=current_stop_called, stop_called=current_stop_called)
+            self._logger.info("Plugin {}: alpha = {}, target = {}, stop_called = {}".format(current_name,
```

Comment:
```suggestion
            message = f'Plugin {current_name}: alpha = {current_alpha}, target = {current_target}'
            if current_stop_called is not None:
                message += f', stop_called = {current_stop_called}'
            self._logger.info(message)
```

Reply:
Resolving in commit

Reply:
Am I removing
```python
                                                                                           current_alpha,
                                                                                           current_target,
                                                                                            current_stop_called))
```
after ` self._logger.info(message)`?

Reply:
Right. I just couldn't figure out how to make a suggested change apply to multiple lines.